src/main/java/org/overture/ego/controller/AuthController.java

```diff
@@ -50,12 +49,18 @@
   @Autowired
   FacebookTokenService facebookTokenService;
 
+  // -- DEMO PROFILE CONFIGURATIONS --
+  @Value("${demo.user.save:true}")
```

Comment:
Could we have just one property called demo

demo will be either true or false